### ISSUE TITLE
/appinfo/app.php is deprecated

### ISSUE THREAD

**Issue by shaynethiessen** [nextcloud/contacts#1831]
Debug logs keep showing the following for the contacts app.

`/appinfo/app.php is deprecated, use \OCP\AppFramework\Bootstrap\IBootstrap on the application class instead.`

This is showing up in the logs every 5ish seconds.

**Comment by skjnldsv** [comment]
Yes, this is a nc20 feature only, until the min supported version of contacts is 20, this will be there.
Keep in mind that yu should not use debug in production, that is not the purpose of it :)

**Comment by shaynethiessen** [comment]
Ah, okay, Thank you for your explanation.

**Comment by skjnldsv** [comment]
You're very welcome :)
Thank you for your interest! :rocket:

**Comment by ChristophWurst** [comment]
The app now support 20+. This is now doable. More details about the process are at https://docs.nextcloud.com/server/stable/developer_manual/app_development/bootstrap.html#bootstrapping-process.
